culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+substantial research, design, and implementation effort. Some of this
+effort cannot even begin until we have a fairly stable understanding of
+other parts of the pipeline. It doesn’t make sense to discuss defenses
+until we’ve defined the protocol in which everyone is to operate.
+
+Furthermore, while the effort required to improve gaming resistance will
+surely bring us benefits in the long term, those benefits only
+materialize when they actually prevent gaming, which can in turn only
+occur if people are trying to game the system in ways that our existing
+systems cannot sufficiently mitigate. Thus, as long as our community
+remains small enough that manual chastising and blacklisting can rein in
+the renegades, we’ll receive no short-term reward for any effort spent
+on this. We have *plenty* of other initiatives into which we’d prefer to
+invest our energy.
+
+Therefore, we propose that we change our approach to trust over time,
+and lay out a plan to do so.
+
+# Trust levels defined
+
+We propose three trust levels, which characterize not the technical
+systems built by the SourceCred contributors but rather the composition
+of a community:
+
+  - **Trust level 3** (TL3): Collaborative. It is taken as [common
+    knowledge][common-knowledge] that everyone in the community is
+    acting in good faith. People feel comfortable reflecting on their
+    own incentives: “am I subconsciously skewing my ‘votes’ to benefit
+    myself?”; “how have my motivations for votes, etc. changed over
+    time?”. When asking the same questions of others, we trust that the
+    responses are thoughtfully and truthfully given. Discussions of
+    accidental or subconscious gaming are treated as precious data to be
+    shared and analyzed, never as evidence of any kind of wrongdoing.
+
+  - **Trust level 2** (TL2): Wary. While the community’s core backbone
+    of contributors are still dedicated to the original ideals, perhaps
+    some new members are more interested in expanding their share of the
+    pie rather than the pie as a whole. Primarily distinguished from TL3
+    because good faith is no longer a uniform assumption. Expect skewed
+    voting records, underrecognition of others’ help, quarrels over
+    attribution or allocation.
+
+  - **Trust level 1** (TL1): Alert. This community controls a scarce
+    resource. That resource is considered intrinsically valuable by some
+    people outside the community, who are willing to devote their time
+    to acquiring that resource without much thought for scruples.
+    Primarily distinguished from TL2 because gaming may now be the main
+    goal rather than an opportunistic “side hustle” on legitimate
+    contributions—noting, of course, that legitimate components may yet
+    be a key component of effective gaming strategies! Expect voting
+    cliques, [sybils], chains of obfuscated self-dealing, attempts to
+    corrupt structures of leadership and governance.
```

Comment:
Link here for "sybils" not defined. I think I'm having a hard time understanding how the trust levels map to tangible permissions / actions. From reading, I'm trying to figure out via "Wary" and "Alert" if Level 1 is the most trusted or if it's 3. It doesn't seem super obvious / is a little confusing so we could probably improve on this, either with better description, name, or even colors.

Reply:
I think "sybils" and mapping to permissions/actions will be familiar to crypto peeps. But probably not to most OSS projects outside of that.